lib/BambuserAPI.js

```diff
@@ -181,6 +181,17 @@ class BambuserAPI {
         } catch (err) {
           handleAPIError(err);
         }
+      },
+      getDownloadLink: async broadcastId => {
+        if (typeof broadcastId !== 'string' || !broadcastId) {
+          throw new Error('Invalid argument. Expected broadcast id.');
+        }
+
+        try {
+          await this.agent.get(`/broadcasts/${broadcastId}/downloads`);
```

Comment:
You probably want to return something here.

Reply:
Indeed, little forgetfulness